src/main/java/com/footballteam/homesite/dao/HomesiteDAO.java

```diff
@@ -0,0 +1,35 @@
+package com.footballteam.homesite.dao;
+
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.footballteam.homesite.model.News;
+
+@Repository
+public class HomesiteDAO {
+
+	@PersistenceContext
+	EntityManager entityManager;
+
+	@Transactional
+	public void editNews(News news) {
+		entityManager.merge(news);
+	}
+
+	public News getNews(int id) {
+		return entityManager.find(News.class, id);
+	}
+	
+	public List<News> getAllNews(){
+		Query query = entityManager.createQuery("SELECT n FROM News n");
```

Comment:
Instead  of writing SQL on your own you can user JPA repository like here:
https://github.com/MichalBojes/footballteam/tree/gpiwosz-dev
it have ale lot of predifined methods, you can also create new ones by typing their names IDE should help with possible ones